repo2docker/buildpacks/conda.py

```diff
@@ -0,0 +1,49 @@
+"""
+Generates a variety of Dockerfiles based on an input matrix
+"""
```

Comment:
Can we ditch this doc string from all the files and instead have one on each class that says something about that specific build pack? Maybe mention the input file it looks for.

Reply:
Can I do that in a different PR? This PR is splitting files, so I don't wanna do much other changes here, since it's hard to see the diff. Also I wanna get this one merged asap to reduce merge conflicts :)